pkg/plugins/app_plugin.go

```diff
@@ -116,7 +116,7 @@ func (app *AppPlugin) InitApp(panels map[string]*PanelPlugin, dataSources map[st
 			app.DefaultNavUrl = cfg.AppSubURL + "/plugins/" + app.Id + "/page/" + include.Slug
 		}
 		if include.Type == "dashboard" && include.DefaultNav {
-			app.DefaultNavUrl = cfg.AppSubURL + "/dashboard/db/" + include.Slug
+			app.DefaultNavUrl = cfg.AppSubURL + "/d/" + include.GetSlugOrUid()
```

Comment:
this wont work now for old slug dashboards

Reply:
Moved the link logic to a new function. GetSlugOrUidLink